openedx/openedx-learning
# feat: CRUD API for Taxonomy Tags [FC-0036]

### Description

This PR introduces the CRUD python and REST APIs for interacting with Taxonomy Tags. Creating, updating and deleting them.

### Supporting Information

Related Tickets:
- Closes https://github.com/openedx/modular-learning/issues/136

### Testing Instructions

Make sure all the tests pass, and they cover all the cases.

---
Private-ref: [FAL-3519](https://tasks.opencraft.com/browse/FAL-3519)
